src/account/Persona.tsx

```diff
@@ -19,6 +20,7 @@ import { walletAddressSelector } from 'src/web3/selectors'
 import pjson from '../../package.json'
 
 const TAG = 'PERSONA'
+const E2E_TEMPLATE_ID = 'itmpl_5FYHGGFhdAYvfd7FvSpNADcC'
```

Comment:
Why do we need this? Isn't this the same template as in firebase?

Reply:
Yes it is, but firebase is [not initialized](https://github.com/valora-inc/wallet/blob/main/src/firebase/saga.ts#L51) in e2e tests so I manually set the template id.

Reply:
hmm. personally not a fan of having e2e specific stuff in components, especially given that this has to be in sync with what we set in firebase. Guess we can leave this as is for now and revisit if this becomes more dynamic (e.g., different templates for different kyc requirements)

Reply:
perhaps a template ID getter could be put in the firebase or config module instead?